bigquery/system-test/tables.test.js

```diff
@@ -0,0 +1,79 @@
+// Copyright 2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+var uuid = require('node-uuid');
+var generateUuid = function () {
+  return 'nodejs_docs_samples_' + uuid.v4().replace(/-/gi, '_');
+};
+var example = require('../tables');
+var options = {
+  bucket: generateUuid(),
+  file: 'data.json',
+  dataset: generateUuid(),
+  table: generateUuid()
+};
+var BigQuery = require('@google-cloud/bigquery');
+var bigquery = BigQuery();
+var Storage = require('@google-cloud/storage');
+var storage = Storage();
```

Comment:
Should these be compressed, i.e `var bigquery = require('@google-cloud/bigquery')();`?